pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -547,4 +547,10 @@ stdenv.mkDerivation ({
 // optionalAttrs (!stripped || cross != null) { dontStrip = true; NIX_STRIP_DEBUG = 0; }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
+
+# This is only here because it was here in previous commits and we
+# want to avoid a meaningless mass rebuild.  These variables get
+# overridden via crossAttrs in an actual cross-build.
+// { crossMingw = false; crossStageStatic = true; }
+
```

Comment:
It's only needed to make my [outside-of-tree tests](https://gist.github.com/DavidEGrayson/8d61032e523b3bc8bbca0c4b281afb52) pass and to avoid a mass rebuild.  The tests in the tree would not be able to detect that a mass rebuild is happening because they only look at the current commit of nixpkgs.  The tests in the tree just make sure that a few native derivations don't get changed when crossSystem is specified.